README.md

```diff
@@ -77,16 +98,44 @@ module Part_2 = struct
 end
 ```
 
-> [!TIP] 
-> It's also helpful to add a `.mli` file, which gives the compiler more 
+#### ReasonML
+
+`lib/problems/problem_2022_01.re`;
+
+```reason
+let year = 2022;
+let day = 1;
+
+module Part_1 = {
+  let run = (input: string): result(string, string) => Ok(input);
+};
+
+module Part_2 = {
+  let run = (input: string): result(string, string) => Ok(input);
+};
+```
+
+> [!TIP]
+> It's also helpful to add a `.mli` or `.rei` file, which gives the compiler more
 > information about which parts of your code are unused and can therefore be
 > cleaned up:
 
+### OCaml
+
 `lib/problems/problem_2022_01.mli`:
+
 ```ocaml_interface
 include Problem.T
 ```
 
+### ReasonML
+
+`lib/problems/problem_2022_01.rei`:
+
+```reason_interface
+include Problem.T;
+```
```

Comment:
**Nit (driveby):** I've been meaning to change this -- I don't think GitHub recognizes `ocaml_interface` (and I bet it doesn't recognize `reason_interface`, either...). Would you mind changing these to `ocaml` and `reason`, respectively?

Reply:
You got it